Added another usage hint to foresee #1072 (#1089)
actions/cache: tips-and-workarounds.md

## Cross OS cache
From `v3.2.3` cache is cross-os compatible when `enableCrossOsArchive` input is passed as true. This means that a cache created on `ubuntu-latest` or `mac-latest` can be used by `windows-latest` and vice versa, provided the workflow which runs on `windows-latest` have input `enableCrossOsArchive` as true. This is useful to cache dependencies which are independent of the runner platform. This will help reduce the consumption of the cache quota and help build for multiple platforms from the same cache. Things to keep in mind while using this feature:
- Only cache those files which are compatible across OSs.
- Caching symlinks might cause issues while restoration as they work differently on different OSs.

> - Only cache files, which are within your github workspace directory. Don't try to use directory pointer like `${{ github.workspace }}` or `~` (home) eventually being evaluated to an absolute path, as the paths will not match the setup across OSs.

Reviewer: ```suggestion
- Only cache files from within your github workspace directory. 
- Avoid using directory pointers such as `${{ github.workspace }}` or `~` (home) which eventually evaluate to an absolute path and will not match across OSs.
```

Reviewer: Minor suggestions.
